### PR TITLE
Updated disaster_recovery.rst

### PR DESCRIPTION
**Pull request by simskij**
I finally got around to continue on the disaster recovery section. Yay!
Added some notes on prioritizing.

### CODE REVIEW

**Comment by miketheman** [comment]
@simskij Thanks! Why the rename of the servers?

**Comment by miketheman** [comment]
@simskij Also, please follow the Style Guide as laid out here: http://www.opsschool.org/en/latest/meta/style_guide.html
Specifically, sentences on new lines.

File: disaster_recovery.rst [hunk 1]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+* sql-srv-001
+* stg-srv-001
+* app-srv-001
 
-.. TODO:: text - How to prioritize recovery components, shared resources, bussiness needs.
+Prioritizing Recovery Components
+--------------------------------
+In the example above, the number of assets to prioritize is low, which might suggest that there is no need for making a priority list.
+However, one thing that you'll learn by doing a couple of disaster recovery excercises is that no matter how small the scope, stakeholders will always try to direct your effors to the assets mosts relevant to them.
+For example, the CEO might want you to prioritize the ERP System over the Domain Controller, which might very well be correct, but as the list of assets grow longer the number of stakeholders wanting to influence your decisions will aswell.
```
Comment (miketheman) [inline disaster_recovery.rst]: "aswell" -> "as well"

File: disaster_recovery.rst [hunk 2]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+
+As you might have guessed, it may be a good idea to actually prioritize your asset list, if not out of necessity then atleast to circumvent any issues that might occur because of differing opinions on what needs to be prioritized.
```
Comment (miketheman) [inline disaster_recovery.rst]: "atleast" -> "at least"

File: disaster_recovery.rst [hunk 3]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+A great way to start out is to create a spreadsheet consisting of the following columns:
+
+* Asset Identification
+    A name, a FQDN or an IP.
+    Whatever helpes you identifying the asset.
```
Comment (miketheman) [inline disaster_recovery.rst]: "helpes" -> "helps"

File: disaster_recovery.rst [hunk 4]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+ 
+* Business Priority
+    Non-Essential, Essential, Critical.
+    This should be decided buy either the board or the senior management team.
```
Comment (miketheman) [inline disaster_recovery.rst]: "buy" -> "by"

**Comment by simskij** [comment]
@miketheman Fixed the new line issue. I named the servers without dashes in my initial pull request, but when i red it now, i figured it might improve the readability if we separate the nouns with dashes in the server names as otherwise it all looks like one long word.

File: disaster_recovery.rst [hunk 5]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+
+* Tiebreaker/Sequential numbering
+    A sequential numbering which will break ties in case multiple assets have the same priority.
+    Should also be decided by the board or the senior management team.
```
Comment (miketheman) [inline disaster_recovery.rst]: What if the person making these lists is the senior management team? Why would the management team matter in sequential numbering?

File: disaster_recovery.rst [hunk 6]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+
+* Business Impact
+    A textual description of what would happen if this asset where to be unavailable.
+
+* Exceptions
+    Any exceptions to the priority above. 
+    A company that is doing billing once every month might not feel that the billing system is critical during any other period then the billing period. 
+    This will of course reflect your real time prioritization if (when) a disaster occurs.
+
+The finished product should, after a signoff from your department manager and the senior management team, be published on your companys intranet, available for anyone.
```
Comment (miketheman) [inline disaster_recovery.rst]: "companys" -> "company's"

File: disaster_recovery.rst [hunk 7]
```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
...
+This is very important as lack of transparency is one of the most common prejudices about IT Depts.
```
Comment (miketheman) [inline disaster_recovery.rst]: "Depts" -> "Departments"

File: disaster_recovery.rst (hunk 5 above)
Comment (simskij) [inline disaster_recovery.rst]: I think I might have to clarify a bit in the text. Think of it like this:

If ABC and XYZ both have the same priority, which one would you like us to prioritize if they both are tied for our attention?

Makes sense?

File: disaster_recovery.rst (hunk 6 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

File: disaster_recovery.rst (hunk 3 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

File: disaster_recovery.rst (hunk 4 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

File: disaster_recovery.rst (hunk 7 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

File: disaster_recovery.rst (hunk 2 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

File: disaster_recovery.rst (hunk 1 above)
Comment (simskij) [inline disaster_recovery.rst]: fixed

**Comment by miketheman** [comment]
Thanks!

**Pull request merged**